staff/gianluca-bragaglia/app-template/about.html

```diff
@@ -0,0 +1,71 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>About</title>
+</head>
+<body>
+    <header>
+        <h1>My App</h1>
+    </header>
+    <nav>
+        <li><a href="home.html">Home</a></li>  
+        <li><a href="news.html">News</a></li> 
+        <li><a href="about.html">About</a></li> 
+        <li><a href="contact.html">Contact</a></li> 
+        <li><a href="profile.html">Profile</a></li> 
```

Comment:
the ```<li>``` tags cannot be used  standalone, may you need to integrate them inside a ```<ul>```

Reply:
fixed